epochStart/metachain/trigger.go

```diff
@@ -11,11 +11,13 @@ import (
 	"github.com/ElrondNetwork/elrond-go/data/block"
 	"github.com/ElrondNetwork/elrond-go/dataRetriever"
 	"github.com/ElrondNetwork/elrond-go/epochStart"
+	"github.com/ElrondNetwork/elrond-go/logger"
 	"github.com/ElrondNetwork/elrond-go/marshal"
 	"github.com/ElrondNetwork/elrond-go/storage"
-	"github.com/prometheus/common/log"
 )
 
+var log = logger.GetOrCreate("epochStart")
```

Comment:
maybe `epochStart/metachain`. Also change for the shard trigger's logger. Right now you have the same logger key for both packages.

Reply:
this is a small  component. there is no need to declare separately for subpackages.

Reply:
yes, the logger should be separated on each package and subpackage

Reply:
done.